### PR TITLE
Create Identity Server BOSH Release for Deployment Pattern 1

### PR DESCRIPTION
**Pull request by chirangaalwis**

### CODE REVIEW

File: bosh-release/deploy.sh [hunk 1]
```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
```
Comment (imesh) [inline bosh-release/deploy.sh]: Is this file being removed and added again as a new file to pattern-1/bosh-release/deploy.sh?

File: bosh-release/jobs/wso2is/monit [hunk 2]
```diff
@@ -1,7 +0,0 @@
-check process wso2is
```
Comment (imesh) [inline bosh-release/jobs/wso2is/monit]: What would the reason for removing this file?

File: bosh-release/jobs/wso2is/templates/config/registry.xml [hunk 3]
```diff
@@ -1,182 +0,0 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
```
Comment (imesh) [inline bosh-release/jobs/wso2is/templates/config/registry.xml]: What would be the reason for removing this file?

File: bosh-release/packages/mysqldriver/packaging [hunk 4]
```diff
@@ -1,13 +0,0 @@
-# abort script on any command that exit with a non zero value
```
Comment (imesh) [inline bosh-release/packages/mysqldriver/packaging]: What would be the reason for removing this file?

File: bosh-release/setup.sh [hunk 5]
```diff
@@ -1,314 +0,0 @@
-#!/bin/bash
```
Comment (imesh) [inline bosh-release/setup.sh]: What would be the reason for removing this file?

File: pattern-1/bosh-release/jobs/wso2is_2/templates/ctl.erb [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+RUN_DIR=/var/vcap/sys/run/wso2is
+LOG_DIR=/var/vcap/sys/log/wso2is
+PIDFILE=${RUN_DIR}/pid
+
+mkdir -p $RUN_DIR $LOG_DIR
+chown -R vcap:vcap $RUN_DIR $LOG_DIR
+
+mkdir -p /var/vcap/store/wso2is/data
+export WSO2_IS_DATA_DIR=/var/vcap/store/wso2is/data/wso2is
```
Comment (imesh) [inline pattern-1/bosh-release/jobs/wso2is_2/templates/ctl.erb]: What would be the usage of WSO2_IS_DATA_DIR?

File: pattern-1/bosh-release/undeploy.sh [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+        --vars-store vbox/creds.yml \
+        -v director_name="Bosh Lite Director" \
+        -v internal_ip=192.168.50.6 \
+        -v internal_gw=192.168.50.1 \
+        -v internal_cidr=192.168.50.0/24 \
+        -v outbound_network_name=NatNetwork
+fi
+
+# remove the added route
+sudo route delete -net 10.244.0.0/16 gw 192.168.50.6
```
Comment (imesh) [inline pattern-1/bosh-release/undeploy.sh]: This command may not work on OSX. Might need to check the OS as we have done in the deploy.sh.

File: bosh-release/deploy.sh (hunk 1 above)
Comment (chirangaalwis) [reply bosh-release/deploy.sh]: @imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

File: bosh-release/jobs/wso2is/monit (hunk 2 above)
Comment (chirangaalwis) [reply bosh-release/jobs/wso2is/monit]: @imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

File: bosh-release/jobs/wso2is/templates/config/registry.xml (hunk 3 above)
Comment (chirangaalwis) [reply bosh-release/jobs/wso2is/templates/config/registry.xml]: @imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

File: bosh-release/packages/mysqldriver/packaging (hunk 4 above)
Comment (chirangaalwis) [reply bosh-release/packages/mysqldriver/packaging]: @imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

File: bosh-release/setup.sh (hunk 5 above)
Comment (chirangaalwis) [reply bosh-release/setup.sh]: @imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

File: bosh-release/deploy.sh (hunk 1 above)
Comment (imesh) [reply bosh-release/deploy.sh]: The PR shows some files have been moved but some as deleted and added as new.

File: bosh-release/jobs/wso2is/monit (hunk 2 above)
Comment (imesh) [reply bosh-release/jobs/wso2is/monit]: The PR shows some files have been moved but some as deleted and added as new.

File: bosh-release/jobs/wso2is/templates/config/registry.xml (hunk 3 above)
Comment (imesh) [reply bosh-release/jobs/wso2is/templates/config/registry.xml]: The PR shows some files have been moved but some as deleted and added as new.

File: bosh-release/packages/mysqldriver/packaging (hunk 4 above)
Comment (imesh) [reply bosh-release/packages/mysqldriver/packaging]: The PR shows some files have been moved but some as deleted and added as new.

File: bosh-release/setup.sh (hunk 5 above)
Comment (imesh) [reply bosh-release/setup.sh]: The PR shows some files have been moved but some as deleted and added as new.

File: pattern-1/bosh-release/jobs/wso2is_2/templates/ctl.erb (hunk 6 above)
Comment (chirangaalwis) [reply pattern-1/bosh-release/jobs/wso2is_2/templates/ctl.erb]: @imesh after tests, the usage of `WSO2_IS_DATA_DIR` was identified as unnecessary. These sections are available here, as I have reused the existing implementation. I have refined the control scripts and rest of the implementation as part of improvements and will be available in future PRs.

**Pull request merged**